ql/src/java/org/apache/hadoop/hive/ql/session/SessionState.java

```diff
@@ -278,11 +278,6 @@
    */
   private Map<URI, HadoopShims.HdfsEncryptionShim> hdfsEncryptionShims = Maps.newHashMap();
 
-  /**
-   * Cache for Erasure Coding shims.
-   */
-  private Map<URI, HadoopShims.HdfsErasureCodingShim> erasureCodingShims;
```

Comment:
I did not purposely remove this, I think it was due to me moving around things. But it does seem valid to remove it ass I can not find any usage of this anywhere.